package.json

```diff
@@ -4,29 +4,32 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "prebuild": "npm run clean && echo \"building...\"",
```

Comment:
Do we want to add linting to help with future contributions?

```
"prebuild": "npm run clean && npm run lint && echo \"building...\"",

"lint": "eslint . --ext .ts",
"lint:fix": "eslint . --ext .ts --fix",
```

Reply:
how would that work, would someone need to run it manually?

Reply:
Alright seems good to me. I can try to add that and ask for a review again.